lib/pci_caps.c

```diff
@@ -276,9 +276,14 @@ handle_px_pxdc_write(vfu_ctx_t *vfu_ctx, struct pxcap *px,
         vfu_log(vfu_ctx, LOG_DEBUG, "MRRS set to %d", p->mrrs);
     }
 
-    if (p->iflr) {
+    if (p->iflr && vfu_ctx->reset) {
```

Comment:
How do we handle the case where `iflr` is written to but `vfu_ctx->reset == NULL`? Shall we at least log something?

Reply:
yeah, this would be an API user error, so it should log a warning at least.
Perhaps we should add to the reset docs that if they set flrc, they must handle that type of reset CB?

Reply:
IMO that should be device implementation bug, as it advertises flrc but did not implemented it.   Added error message for such cases.